bindings_test.go

```diff
@@ -382,6 +383,62 @@ func TestBindingInterfaceString(t *testing.T) {
 	})
 }
 
+func TestBulkArrayBindingUUID(t *testing.T) {
+	max := math.Pow10(5) // 100K because my power is maximum
+	uuids := make([]any, int(max))
+
+	createTable := "CREATE OR REPLACE TABLE TEST_PREP_STATEMENT (id INT autoincrement start 1 increment 1, uuid VARCHAR)"
+	insert := "INSERT INTO TEST_PREP_STATEMENT (uuid) VALUES (?)"
+
+	for i := range uuids {
+		uuids[i] = newTestUUID()
+	}
+
+	runDBTest(t, func(dbt *DBTest) {
+		var rows *RowsExtended
+		t.Cleanup(func() {
+			if rows != nil {
+				assertNilF(t, rows.Close())
+			}
+
+			dbt.exec(deleteTableSQL)
+		})
+
+		dbt.mustExec(createTable)
+
+		bound := Array(&uuids)
+		res := dbt.mustExec(insert, bound)
+
+		if affected, _ := res.RowsAffected(); affected != int64(max) {
+			t.Fatalf("failed to insert all rows. expected: %f.0, got: %v", max, affected)
+		}
+
+		rows = dbt.mustQuery("SELECT * FROM TEST_PREP_STATEMENT ORDER BY ID")
+
+		for i := 0; rows.Next(); i++ {
+			var (
+				id  int
+				out testUUID
+			)
+			if err := rows.Scan(&id, &out); err != nil {
+				t.Fatal(err)
+			}
+
+			var found bool
+			for _, u := range uuids {
```

Comment:
This makes it `O(n^2)` complexity :) Two suggestions to fix this - either change the table to contain incremented value and order by id instead of UUIDs or sort `uuids` array :)

Reply:
This is fixed @sfc-gh-pfus